pubspec.yaml

```diff
@@ -70,4 +70,6 @@ flutter:
   fonts:
     - family: neuzeit
       fonts:
-        - asset: assets/fonts/neuzeit.otf
+        - asset: assets/fonts/neuzeit_book.otf
```

Comment:
What about the `weight`(400) for the normal style.

Reply:
@Wadeewee do we really need it because 400 is the default weight of the normal font? 🤔

Reply:
@AVI5HEK, Is it rendered properly without defining the weight(400) for the normal? If yes, just skip it then

Reply:
@Wadeewee yes, no difference.